ansible/playbooks/service/discourse.yml

```diff
@@ -0,0 +1,84 @@
+---
+
+- name: Manage Discourse service
+  hosts: [ 'debops_service_discourse' ]
+  become: True
+
+  environment: '{{ inventory__environment | d({})
+                   | combine(inventory__group_environment | d({}))
+                   | combine(inventory__host_environment  | d({})) }}'
+
+  roles:
+
```

Comment:
You are missing the `debops.keyring` role used by other roles to manage GPG keys and APT repositories. You should include it near the top of the playbook.